src/app/views/cheat-sheets/train-colors/train-colors.component.html

```diff
@@ -3,11 +3,11 @@
     <table class="table table-sm table-hover fixed-width">
         <caption class="text-center">
             <app-blueprint-clipboard [url]="sheetData?.blueprint"></app-blueprint-clipboard>
-            Suggested locomotive colors based on cargo
+            Locomotive colors are based on the color of the cargo.
```

Comment:
This is a table title, i think its fine as it is.

Reply:
It's gone now.